plugins/uefi/fu-plugin-uefi.c

```diff
@@ -58,6 +58,9 @@ fu_plugin_init (FuPlugin *plugin)
 	fu_plugin_add_rule (plugin, FU_PLUGIN_RULE_RUN_AFTER, "upower");
 	fu_plugin_add_report_metadata (plugin, "FwupdateVersion", LIBFWUP_LIBRARY_VERSION);
 	fu_plugin_add_report_metadata (plugin, "EfivarVersion", EFIVAR_LIBRARY_VERSION);
+#if LIBFWUP_CHECK_VERSION(12,0)
+	fu_plugin_add_runtime_version (plugin, "com.redhat.fwupdate", fwup_version ());
```

Comment:
As part of this change I think it's worth also comparing the compile time version and runtime version and notating if they're different in `g_debug` output for future debugging of problems.